src/tqec/templates/orchestrator.py

```diff
@@ -313,7 +327,6 @@ def build_array(self, indices_map: tuple[int, ...]) -> numpy.ndarray:
         shape = self._get_shape_from_bounding_box(bbul, bbbr)
 
         ret = numpy.zeros(shape.to_numpy_shape(), dtype=int)
-        # tid: template id
```

Comment:
This comment was here to explain the variable name used in the code and avoid super-long names that would make the code harder to read. Is there a specific reason why you removed it?

Reply:
This was an accident